Monika After Story/game/script-topics.rpy

```diff
@@ -14736,6 +14736,44 @@ label monika_dying_same_day:
 
     return
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_player_away",
+            category=["Monika"],
```

Comment:
```suggestion
            category=["monika"],
```

categories are always lowercased in code. using uppercase actually counts as a different letter and will make a new category